src/Microsoft.DotNet.Interactive.Tests/ProxyKernelConnectionTestsBase.cs

```diff
@@ -133,7 +133,7 @@ public async Task fast_path_commands_over_proxy_can_be_handled()
     {
         var connector = CreateConnector();
```

Comment:
minor nit: Also rename this variable from `connector` to `createKernel` to match the above test?